Documentation/Commands/xcassets.md

```diff
@@ -0,0 +1,25 @@
+# Asset Catalog parser
+
+## Input
+
+The assets parser accepts one (or more) asset catalogs, which it'll parse for supported set types and groups. We currently support the following types:
+- Group Type
```

Comment:
That's what's usually called a "folder" in the asset catalog, right? Might want to mention that in parentheses maybe